spec/rubocop/cop/lint/deprecated_open_ssl_constant_spec.rb

```diff
@@ -66,6 +66,25 @@
     RUBY
   end
 
+  it 'does not register an offense with cipher constant and argument is a variable' do
+    expect_no_offenses(<<~RUBY)
+      mode = "cbc"
+      OpenSSL::Cipher::AES128.new(mode)
+    RUBY
+  end
+
+  it 'does not register an offense with cipher constant and send argument is a method' do
+    expect_no_offenses(<<~RUBY)
+      OpenSSL::Cipher::AES128.new(do_something)
+    RUBY
+  end
+
+  it 'does not register an offense with cipher constant and argument is a consntant' do
```

Comment:
Found a typo — I hope that this kind of drive-by review comment is OK to do and feels OK to you, @koic :heart:

```suggestion
  it 'does not register an offense with cipher constant and argument is a constant' do
```

Reply:
Oops! Thank you for your review :-)